### DISCUSSION TITLE
SIP-0002 discussion

### DISCUSSION THREAD

**Discussion by Semisol** [stackchainsats/stackchain#3] [General]
See: https://github.com/derekross/stackchain/blob/main/sips/sip-0002.txt
Proposer: @Semisol 
Twitter thread: https://twitter.com/Semisol_Public/status/1549872722368397312

**Comment by derekross** [comment]
I like how you included NEWSTACK just in case we get SIP-0001 approved @Semisol

**Comment by DanjaminDW** [comment]
Additional Stackpool should be replied to underneath the original Stackpool claim. When enough funds are had to create the next block the last Stackpool combines all previous amounts in one tweet under the highest stack height.

If a new stack height is created during a Stackpool process users should resubmit their stack underneath the new highest Stack.

Also I believe Stackjoin is a better term than Stackpool.

**Reply by Semisol** [reply]
Was thinking Stackjoin would be the merging operation -- say you had an orphaned Stack and wanted to place it on tip. As long as the orphaned Stack is not on an active fork you can Stackjoin it with another purchase

**Reply by DanjaminDW** [reply]
This would help with not needing to resubmit your stack but how does one know if a stack was previously used or not?

**Reply by Semisol** [reply]
> This would help with not needing to resubmit your stack but how does one know if a stack was previously used or not?

Comparing images, and partly trust.
If we want to eliminate trust, we may want to link to the orphaned tweet and the merged purchase to make the Stackjoin

**Reply by DanjaminDW** [reply]
I can see that working. There's enough people to respond to the Stackjoin to validate there wasn't a 'double spend'.

**Reply by derekross** [reply]
Agree, we have enough validators watching that thread. Anyone on the thread would be able to respond and prevent a double spend.

**Comment by Semisol** [comment]
This proposal has been superseded by SIP-0003
